project/build.scala

```diff
@@ -5,17 +5,22 @@ import scala.util.Properties
 // sbt-assembly
```

Comment:
Going to look into doing this here -- sbt 0.13.16 shows a warning about using `build.scala`, but the project still compiles. If moving to `build.sbt`turns out to be really involved I'll spin it off into another card.